diff --git a/gcc/cp/Make-lang.in b/gcc/cp/Make-lang.in
index 23d98c897b5..ef598f16c49 100644
--- a/gcc/cp/Make-lang.in
+++ b/gcc/cp/Make-lang.in
@@ -59,10 +59,8 @@ CFLAGS-cp/module.o += -DHOST_MACHINE=\"$(host)\" \
 
 # In non-release builds, use a date-related module version.
 ifneq ($(DEVPHASE_c),)
-# Some date's don't grok 'r', if so, simply use today's
-# date (don't bootstrap at midnight).
-MODULE_VERSION := $(shell date -r $(srcdir)/cp/module.cc '+%y%m%d-%H%M' \
-  2>/dev/null || date '+%y%m%d-0000' 2>/dev/null || echo 0)
+
+MODULE_VERSION := 220101-0000
 
 CFLAGS-cp/module.o += -DMODULE_VERSION='($(subst -,,$(MODULE_VERSION))U)'
 endif
